types/externs.d.ts

```diff
@@ -143,31 +147,35 @@ declare global {
     export interface CliFlags extends Flags {
       _: string[];
       chromeFlags: string;
-      // Output path for the generated assets
+      /** Output path for the generated results. */
       outputPath: string;
-      // Flag to save the trace contents and screenshots to disk
+      /** Flag to save the trace contents and screenshots to disk. */
       saveAssets: boolean;
-      // Flag to open the report immediately
+      /** Flag to open the report immediately. */
       view: boolean;
-      // Enables error reporting
+      /** Flag to enable error reporting. */
       enableErrorReporting?: boolean;
-      // Flag to print out a list of all audits + categories
+      /** Flag to print a list of all audits + categories. */
       listAllAudits: boolean;
-      // Flag to print out a list of all required trace categories
+      /** Flag to print a list of all required trace categories. */
       listTraceCategories: boolean;
-      // A preset audit of selected audit categories to run
+      /** A preset audit of selected audit categories to run. */
       preset?: 'full'|'mixed-content'|'perf';
+      /** A flag to enable logLevel 'verbose'. */
       verbose: boolean;
+      /** A flag to enable logLevel 'silent'. */
       quiet: boolean;
-      // following are given defaults in cli-flags, so are not optional like in Flags or SharedFlagsSettings
-      output: OutputMode[];
-      port: number;
-      hostname: string;
+      /** A flag to print the normalized config for the given config and options, then exit. */
       printConfig: boolean;
-      // Path to the file where precomputed lantern data should be read from.
+      /** Path to the file where precomputed lantern data should be read from. */
       precomputedLanternDataPath?: string;
-      // Path to the file where precomputed lantern data should be written to.
+      /** Path to the file where precomputed lantern data should be written to. */
       lanternDataOutputPath?: string;
+
+      // The following are given defaults in cli-flags, so are not optional like in Flags or SharedFlagsSettings.
```

Comment:
Should this also be `/** */`?

Reply:
> Or is it just // b/c it is a comment instead of an annotation for a single field?

exactly. It's an explanation for why the properties below that point are different (those three properties also have doc strings from their base definitions in `SharedFlagsSettings` and `Flags`)